addon/components/file-picker.js

```diff
@@ -166,7 +169,13 @@ export default Component.extend({
       };
 
       reader.onprogress = Ember.run.bind(this, (event) => {
-        this.set('progressValue', event.loaded / event.total * 100);
+        let val;
+        if (event.total === 0) {
+          val = null;
+        } else {
+          val = event.loaded / event.total * 100;
+        }
```

Comment:
Some browsers (at least chromium) throws if `value` of `progress` element is `NaN`. Since `event.total` could be zero on init and division by zero is NaN we have to prevent this.

Old implementation prevents this falsy check of `progressValue` in `progressStyle`: `var width = this.get('progressValue') || 0;`. https://github.com/jelhan/ember-cli-file-picker/blob/fix-csp/addon/components/file-picker.js#L35

https://chromium.googlesource.com/chromium/blink/+/ba8502b7f4107fa57f43f3d41df8e55cb4453e01/LayoutTests/fast/dom/HTMLProgressElement/set-progress-properties-expected.txt